server/index.js

```diff
@@ -54,25 +45,14 @@ app.webpackConfig = webpackConfig;
 
 app.init = function({
   logErrors,
-  peers = process.env.CADENCE_TCHANNEL_PEERS || PEERS_DEFAULT,
-  retryFlags = REQUEST_RETRY_FLAGS_DEFAULT,
-  retryLimit = process.env.CADENCE_TCHANNEL_RETRY_LIMIT ||
-    REQUEST_RETRY_LIMIT_DEFAULT,
-  serviceName = process.env.CADENCE_TCHANNEL_SERVICE || SERVICE_NAME_DEFAULT,
-  timeout = REQUEST_TIMEOUT_DEFAULT,
-  transportClientType = process.env.TRANSPORT_CLIENT_TYPE ||
-    TRANSPORT_CLIENT_TYPE_DEFAULT, // 'tchannel', 'grpc'
-  useWebpack = process.env.NODE_ENV !== 'production',
-  enableAuth = process.env.ENABLE_AUTH === 'true',
-  authType = process.env.AUTH_TYPE,
-  authAdminJwtPrivateKey = process.env.AUTH_ADMIN_JWT_PRIVATE_KEY,
+  peers = config.peers,
+  requestConfig = config.requestConfig,
```

Comment:
Actually, this seems to be a breaking change:
Removing retryFlags, retryLimit ... and replacing them with `requestConfig`.
The Init function is exposed, and this can break current usages.
https://www.npmjs.com/package/cadence-web#initoptions

Reply:
yep, seems so. Fixing